contracts/examples/DXswapRelayer.sol

```diff
@@ -0,0 +1,325 @@
+pragma solidity =0.6.6;
+
+import './OracleCreator.sol';
+import './../interfaces/IDXswapFactory.sol';
+import './../interfaces/IDXswapRouter.sol';
+import './../libraries/TransferHelper.sol';
+import './../interfaces/IERC20.sol';
+import './../libraries/SafeMath.sol';
+
+contract DXswapRelayer {
+    using FixedPoint for *;
+    using SafeMath for uint256;
+
+    event Pool(
+        address _tokenA,
+        address _tokenB,
+        uint256 _amountTokenA,
+        uint256 _amountTokenB,
+        uint256 _minA,
+        uint256 _minB,
+        uint256 _liquidity
+    );
+
+    struct Commitment {
+        uint8 action; /* 1=liquidity provision; 2=liquidity removal; 3=swap */
+        address tokenA;
+        address tokenB;
+        uint256 amountA;
+        uint256 amountB;
+        uint256 priceTolerance;
+        uint256 minReserveA;
+        uint256 minReserveB;
+        uint256 deadline;
+        uint8 maxWindows;
+        uint256 executionBounty;
+        uint256 oracleId;
+        bool executed;
+    }
+
+    address payable public immutable dxdaoAvatar;
```

Comment:
Lets use owner instead of dxdao, we want to allow this to be used anywhere, and using specific names to our case might get people confused, is better to read it and undertsand the owner than dxdao, someone who does not know what dxdao is will get confused.